frontend/src/PatientApp.tsx

```diff
@@ -29,6 +27,20 @@ const styles = (theme: Theme) =>
     },
     title: {
       flexGrow: 1,
+      fontFmily: "Roboto",
```

Comment:
not sure we should be switching font family as Susanne is not designing for style right now but more for function.

Reply:
I just pulled the styles from Susanne's mock.